### PR TITLE
Increase MaxReturned for polling inventory products from 50 to 500

### PR DESCRIPTION
**Pull request by raysapida**

### CODE REVIEW

**Review by gorandev — APPROVED** [review]

**Comment by msalo3** [comment]
This looks fine to me - however, maybe it would be worth it to default to 50 and then use a config variable to set it to something else? `config[:max_returned] = 500`?

**Comment by msalo3** [comment]
That way we can keep functionality as is and also re-use the "grab more" functionality later.

**Comment by raysapida** [comment]
Yeah, I think that would be great, I'll add that in now!

File: lib/qbwc/request/products.rb [hunk 1]
```diff
@@ -143,9 +143,13 @@ def polling_current_items_xml(params, config)
             return build_polling_from_config_param(params, session_id, time)
           end
 
+          if params['quickbooks_max_returned'] && params['quickbooks_max_returned'] != ""
+            inventory_max_returned = params['quickbooks_max_returned']
```
Comment (msalo3) [inline lib/qbwc/request/products.rb:148]: .to_i? I don't know if it's needed here or not?

Comment (msalo3) [reply lib/qbwc/request/products.rb:148]: It's all a string I guess right?

**Review by msalo3 — APPROVED** [review]

File: lib/qbwc/request/products.rb (hunk 1 above)
Comment (raysapida) [reply lib/qbwc/request/products.rb:148]: We don't need to_i but this block would probably break now that I think of it due to undefined local variable. I'll push an update

**Pull request merged**